libr/io/io.c

```diff
@@ -312,18 +320,21 @@ R_API bool r_io_vread_at(RIO* io, ut64 vaddr, ut8* buf, int len) {
 	if (!io->maps) {
 		return false;
 	}
-	return onIterMap_wrap (io->maps->tail, io, vaddr, buf, len, R_IO_READ, fd_read_at_wrap, NULL);
+	onIterMap (io->maps->tail, io, vaddr, buf, len, R_IO_READ, fd_read_at_wrap, &ret);
```

Comment:
`bool ret = true; onIterMap (...., &ret); return ret; ` ugly

Your revert is a beauty regression